app/scripts/modules/core/navigation/state.provider.ts

```diff
@@ -17,6 +17,10 @@ export class StateConfigProvider implements IServiceProvider {
   private root: INestedState = {
     name: 'home',
     abstract: true,
+    url: '?{debug:boolean}',
```

Comment:
does this recognize `true`? I was under the impression that `boolean` was `0` or `1`...